### DISCUSSION TITLE
Undefined array key "description"

### DISCUSSION THREAD

**Discussion by ramoneg** [livewire/livewire#4802] [7. Bugs (archived)]
Hello!

We are facing some bugs since updating to the latest minor version of Livewire.
Since this is a closed source repository of ours I cannot share any snippets of our code.

**I forked the repo and added a check, this solved the issue with all the tests passing.**
Check out my commit [here](https://github.com/ramoneg/livewire/commit/a4abdcdabe2c55f9fab108613f38167ce66fa42a)

The situation in a nutshell:

When creating a new resource with **translated fields** of any model we are receiving the following error:

![Livewire Error](https://user-images.githubusercontent.com/59345351/158180587-6cc63af9-fba0-4165-88a9-f8b34816c4d0.png)

The validation rules of the description field are:
`['nullable', 'string']`

Do you have any inputs?
Seems like in my eyes there's a check missing if the given data is not provided at all.

Thank you!

**Comment by joshhanley** [comment]
@ramoneg any chance you could submit a PR with the changes and tests if you think you have found a bug?

Make sure not to submit it from your master branch, but instead use a feature branch as described in our [contribution guide](https://laravel-livewire.com/docs/2.x/contribution-guide).

Thanks!
